### DISCUSSION TITLE
Android Auto in VC

### DISCUSSION THREAD

**Discussion by BooBoo124** [Mr-MIBonk/M.I.B._More-Incredible-Bash#192] [Q&A] [answered]
Hi,
Thanks for all of the great work you do! Will it ever  be possible to show the Google map on the virtual cockpit screen via an android auto connection patched via MIB?

**Comment by harman-f** [comment, marked as answer]
so far no solution is found here.

Audi R8/TT have this due to a differnt concept, all MMI is displayed in VC/AID.
